lib/minify.js

```diff
@@ -0,0 +1,168 @@
+exports.readFile = function() {
+    DefaultsError.croak("readFile not supported");
+};
+
+exports.writeFile = function() {
+    DefaultsError.croak("writeFile not supported");
+};
+
+exports.simple_glob = function(files) {
+    return files;
```

Comment:
Seems the current code handles string as parameter fine, adding test as extra insurance for now.